dashboard/app/assets/javascripts/videos.js.erb

```diff
@@ -135,7 +135,9 @@ function setupVideoFallback(videoInfo, playerWidth, playerHeight, shouldStillAdd
   }
 
   if (videoInfo['force_fallback']) {
-    addFallbackVideoPlayer(videoInfo, playerWidth, playerHeight);
+    setTimeout(function() {
+      addFallbackVideoPlayer(videoInfo, playerWidth, playerHeight);
+    }, 0);
```

Comment:
The added timeouts fix an ordering bug in the `createVideoWithFallback` code path:

`createVideoWithFallback` function returns the newly-created `<iframe id="video"/>` element, which gets appended to the #video-container by the calling script. However, before this function returns `setupVideoFallback` is called, which calls `addFallbackVideoPlayer`, which expects to find the iframe element in the DOM tree. In most cases this worked okay because `onYouTubeBlocked`/`testImageAccess` calls `addFallbackVideoPlayer` asynchronously (so the iframe element existed by the time it was called), but in some cases (e.g., when `fallback_only` or `force_youtube_fallback` are set), the callback would be invoked immediately, causing an error.

Adding these timeouts fixed this error for these edge cases in my manual tests.